tests/aws/services/events/conftest.py

```diff
@@ -66,6 +66,21 @@ def _create_event_bus(**kwargs):
             LOG.warning(f"Failed to delete event bus {event_bus_name}: {e}")
 
 
+@pytest.fixture
+def events_create_default_or_custom_event_bus(events_create_event_bus, region_name, account_id):
+    def _create_default_or_custom_event_bus(event_bus_type: str = "default"):
+        if event_bus_type == "default":
+            event_bus_name = "default"
+            event_bus_arn = f"arn:aws:events:{region_name}:{account_id}:event-bus/default"
+        if event_bus_type == "custom":
+            event_bus_name = f"test-bus-{short_uid()}"
+            response = events_create_event_bus(Name=event_bus_name)
+            event_bus_arn = response["EventBusArn"]
+        return event_bus_name, event_bus_arn
```

Comment:
What happens in the else case?

It's probably better to fail fast here and raise for `else` rather than ignore silently

Reply:
The IDE also warns about this:
<img width="1113" alt="Screenshot 2024-07-12 at 14 20 46" src="https://github.com/user-attachments/assets/c6a40f33-bf18-4edc-8ce9-2e26047982fd">

Reply:
per default, it will be the "default" one since this is the default value set if no value is provided for event_bus_type, this should also avoid that no value is set, so  the IDE warning can be disregarded

Reply:
👍  `else` fixes the warning and using `cutsom` (typo) ;)